Week3/Homework/Js-exercises/cardValidator.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+const cardValidator = function(cardNumber) {
+  //first converts our number into an array of strings then using for of, converts the string array to numbers array
+  if (Number.isInteger(Number(cardNumber)) !== true) {
+    return console.log("invalid characters!");
+  }
+  let strArr = cardNumber.toString().split("");
+  const numArr = [];
+  for (let ele of strArr) {
+    numArr.push(Number(ele));
+  }
+  console.log(numArr.length);
+  if (numArr.length !== 16) {
+    return console.log("invalid card ,there is less or more than 16 digits!");
```

Comment:
I think you meant to return the string and not the console.log operation, right? 🙂